main.css

```diff
@@ -0,0 +1,9 @@
+body {
+  background-color: teal;
+  font-family: arial;
+  text-align: center;
+}
+
+li {
+  list-style: none;
+}
```

Comment:
I was hoping we would apply a bit more style to these before committing. Let's discuss.